x-pack/plugins/timelines/common/utils/accessibility/helpers.ts

```diff
@@ -5,14 +5,14 @@
  * 2.0.
  */
 
-import { DRAGGABLE_KEYBOARD_WRAPPER_CLASS_NAME } from '../drag_and_drop/helpers';
+import React from 'react';
 import {
+  DRAGGABLE_KEYBOARD_WRAPPER_CLASS_NAME,
+  HOVER_ACTIONS_ALWAYS_SHOW_CLASS_NAME,
   NOTES_CONTAINER_CLASS_NAME,
   NOTE_CONTENT_CLASS_NAME,
   ROW_RENDERER_CLASS_NAME,
-} from '../../../timelines/components/timeline/body/helpers';
-import { HOVER_ACTIONS_ALWAYS_SHOW_CLASS_NAME } from '../with_hover_actions';
-
+} from '@kbn/securitysolution-t-grid';
```

Comment:
Would it make sense for these accessibility helper functions to be moved to the `kbn-securitysolution-t-grid` package (for sharing), if this file didn't depend on these class names?

Reply:
These helper functions will get deleted soon after we are using EuiDataGrid. I think that we can move forward the way it is and delete them when we are using the EuiDataGrid.